ion-schema/src/main/kotlin/com/amazon/ionschema/internal/TypeImpl.kt

```diff
@@ -40,6 +41,22 @@ internal class TypeImpl(
     private companion object {
         private val ION = IonSystemBuilder.standard().build()
         private val ANY = ION.newSymbol("any")
+
+        /**
+         * Order in which constraints should be evaluated. Lower is first.
+         *
+         * The [validate] function in this class has
+         */
+        private val CONSTRAINT_EVALUATION_ORDER = mapOf(
+            "annotations" to -1,
+            // By default, all constraints are priority 0
+            "ordered_elements" to 1,
```

Comment:
This could use some comment on why `ordered_elements` is set to 1 and `annotations` to -1.

Reply:
Removed `ordered_elements`. Added a comment about why `annotations` gets special treatment.